nodejs/awsx/ec2/vpc.ts

```diff
@@ -71,12 +71,12 @@ export class Vpc extends pulumi.ComponentResource {
                 enableDnsHostnames: utils.ifUndefined(args.enableDnsHostnames, true),
                 enableDnsSupport: utils.ifUndefined(args.enableDnsSupport, true),
                 instanceTenancy: utils.ifUndefined(args.instanceTenancy, "default"),
-            });
+            }, { parent: this });
             this.id = this.vpc.id;
 
             // Create the appropriate subnets.  Default to a single public and private subnet for each
             // availability zone if none were specified.
-            const topology = new VpcTopology(this, name, cidrBlock, numberOfAvailabilityZones, opts);
+            const topology = new VpcTopology(this, name, cidrBlock, numberOfAvailabilityZones, { parent: this, ...opts });
```

Comment:
`...opts` should be necessary.  we should just need to pass along us as the parent.

Reply:
Fixed. Only `opts` left